drivers/volume/portworx/portworx.go

```diff
@@ -1524,7 +1740,7 @@ func (p *portworx) GetClusterDomains() (*stork_crd.ClusterDomains, error) {
 	return clusterDomainsInfo, nil
 }
 
-func (p *portworx) ActivateClusterDomain(clusterDomainName string) error {
+func (p *portworx) ActivateClusterDomain(cdu *stork_crd.ClusterDomainUpdate, clusterDomainName string) error {
```

Comment:
You don't need to pass in the name if you have the object, you can get it from there. Same for deactivate.

Reply:
Good point. i will fix that